content/outcomes/cloud-native-development-in-practice/_index.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Cloud Native Development in Practice
+weight: -1
+layout: single
+description: |
+    Build a Cloud Native App.
+    Run it on TAS.
+    Scale it on TAS.
+    Then evolve it to Microservices.
+team:
+  - VMware Tanzu Labs
+---
+
+## Why is this learning path important?
+
+This learning path will take you on a journey of building a
+greenfield blocking web application from scratch,
+getting it to run on a modern cloud native platform,
+and how to deal with it after it goes to production.
+
+If you have not yet completed the
+[Cloud Native Application Development with Spring](../cloud-native-development/) learning path,
+complete that first.
+
+**This series takes the concepts you learn in the**
+**[Cloud Native Application Development with Spring](../cloud-native-development/) learning path**
+**and applies them to a specific application context and**
+**codebase.**
+
+In this learning path you will build a greenfield Spring Boot
+application and deploy it on TAS.
+
+Then you will run it and maintain it in production.
+
+You will gain an understanding of how Tanzu Application Service (TAS)
+platform works,
+how its container orchestration compares to Kubernetes,
+and will recap some fundamentals of Spring and Spring Boot along the
+way.
+
+You will also apply some of the previous concepts such as
+[Cloud Native Development](../cloud-native-development/),
+[Test Driven Development](../application-development/test-driven-development/)
+and
+[Continuous Integration and Continuous Delivery](../../guides/ci-cd/ci-cd-what-is/).
+
+## Who is this learning path for?
+
+This learning path is for software engineers and architects who wish to
+gain a deeper understanding of modern Cloud Native Platforms and how
+Spring Boot applications run on them.
+
+## Learning Outcomes
```

Comment:
```suggestion
## Learning outcomes
```
We tend to prefer "sentence case" for headings. You have a bit of a mixture of both here. Consider revising for consistency.